src/components/HomePage/FAQSection.js

```diff
@@ -131,29 +132,56 @@ function FAQSection() {
 		},
 		{
 			question: `What if I don’t have a team?`,
```

Comment:
Our current FAQ uses a combination of `'` and `’`. This is probably the fault of the people who made the website in the previous year. However we want to be as consistent as possible. Can you change all instances of  `’` to `'`?

You may not need to use the `&apos` in the template strings, just a `'` should be enough.